File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/PotionOfRejuvenation.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable;

/* JADX INFO: loaded from: classes3.dex */
public class PotionOfRejuvenation extends Consumable {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable, it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.consumable_potion_of_rejuvenation_name;
        this.idDescription = R.string.consumable_potion_of_rejuvenation_description;
        this.idImage = R.drawable.potion_of_rejuvenation;
        this.price = 26L;
    }

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable
    public int printConsumeImage() {
        return R.drawable.consume_potion_of_rejuvenation;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/PotionOfRejuvenation.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable

class PotionOfRejuvenation : Consumable() {
    override fun configureProperties() {
        idName = R.string.consumable_potion_of_rejuvenation_name
        idDescription = R.string.consumable_potion_of_rejuvenation_description
        idImage = R.drawable.potion_of_rejuvenation
        price = 26L
    }

    override fun printConsumeImage(): Int = R.drawable.consume_potion_of_rejuvenation
}
